test/math/FloatVectorListTest.ooc

```diff
@@ -337,6 +337,29 @@ FloatVectorListTest: class extends Fixture {
 			list free()
 			interpolatedList free()
 		})
+		this add("interpolate~cubicSpline", func {
+			list := FloatVectorList new()
+			list add(2.0f)
+			list add(1.0f)
+			list add(6.0f)
+			list add(4.0f)
+			list add(7.0f)
+			numberOfPointsBetween := 1
+			interpolatedList := list interpolate~cubicSpline(numberOfPointsBetween)
+
+			expect(interpolatedList count, is equal to(list count + numberOfPointsBetween * (list count - 1)))
+			expect(interpolatedList[0], is equal to(list[0]) within(1.0e-5f))
+			expect(interpolatedList[1], is equal to(0.67634f) within(1.0e-5f))
+			expect(interpolatedList[2], is equal to(list[1]) within(1.0e-5f))
+			expect(interpolatedList[3], is equal to(3.72098f) within(1.0e-5f))
+			expect(interpolatedList[4], is equal to(list[2]) within(1.0e-5f))
+			expect(interpolatedList[5], is equal to(5.31473f) within(1.0e-5f))
+			expect(interpolatedList[6], is equal to(list[3]) within(1.0e-5f))
+			expect(interpolatedList[7], is equal to(4.77009f) within(1.0e-5f))
+			expect(interpolatedList[8], is equal to(list[4]) within(1.0e-5f))
```

Comment:
Most other tests have `tolerance := 1.0e-5f`, so for me it's OK to change it here and use `tolerance` to make it easier to read.